packages/fetch-core/src/withFetchRedirectionOn401.tsx

```diff
@@ -0,0 +1,16 @@
+﻿import { wrapAuthenticating } from './withFetchRedirectionOn403';
+
+export const fetchWithRedirectionOn401 = (fetch: typeof window.fetch, history: ReactOidcHistory) => async (
+  url: RequestInfo,
+  options: RequestInit = { method: 'GET' }
+) => {
+  const response = await fetch(url, options);
+  if (response.status === 401) {
+    history.push('/authentication/not-authenticated');
```

Comment:
**issue:** URL should be configurable, also, it may not force redirection is user do not want to, could generate frustration here

Reply:
You can be create issue about it, indeed